File: gigachadsys-model/src/main/java/pe/edu/pucp/gigachadsys/model/Salon.java
package pe.edu.pucp.gigachadsys.model;

public class Salon {
    // Atributos
    private String nobreSalon;
    private int aforoMaximo;

    public int getIdSalon() {
        return idSalon;
    }

    public void setIdSalon(int idSalon) {
        this.idSalon = idSalon;
    }

    private int idSalon;

    //Constructores
    public Salon() {
    }

    public Salon(String nobreSalon, int aforoMaximo) {
        this.nobreSalon = nobreSalon;
        this.aforoMaximo = aforoMaximo;
    }

    public Salon(int idSalon, String nombreSalon, int aforoMaximo) {
    }

    //Setters y Getters
    public String getNombreSalon() {
        return nobreSalon;
    }

    public void setNombreSalon(String nobreSalon) {
        this.nobreSalon = nobreSalon;
    }

    public int getAforoMaximo() {
        return aforoMaximo;
    }

    public void setAforoMaximo(int aforoMaximo) {
        this.aforoMaximo = aforoMaximo;
    }


    //Metodos
	public boolean verificarDisponibilidadAforo(int cantidadActual) {
        return true;
    }

    public void setActivo(boolean activo) {

    }
}

File: C#/GigaChadSys/GigaChadSys-Model/Salon.cs
namespace GigaChadSysModel;

public class Salon
{
    public int IdSalon { get; set; }
    public string NombreSalon { get; set; } = "";
    public int AforoMaximo { get; set; }
    public bool Activo { get; set; }

    public Salon() { }

    public Salon(int idSalon, string nombreSalon, int aforoMaximo, bool activo)
    {
        IdSalon = idSalon;
        NombreSalon = nombreSalon;
        AforoMaximo = aforoMaximo;
        Activo = activo;
    }
}